schedule_lessons/dashboard/static/dashboard/js/agenda.js

```diff
@@ -1,6 +1,32 @@
 $(document).ready(function () {
   $('[data-toggle="tooltip"]').tooltip();
 
+  $(".acknowledge-btn").on('click', function(e) {
+    var data;
```

Comment:
let's use ES6 and change this from `var` to `let`

Reply:
also, does `data` have other properties apart from `show_tutorial`? if not, it looks like the only reason we are using an object here is because of the format it needs to be in for the ajax request. maybe we can just do:

```
let data = { show_tutorial: 0 };

if (!$('#dontShowTutorialCheckbox').is(':checked')) {
    data.show_tutorial = 1;
}
```

and that way maybe we can even get rid of the else statement?

Reply:
nit: also, any particular reason we are using `0` and `1` instead of directly using `false` and `true` respectively? It would be a bit easier to read if we used the words I think and we wouldn't need the comments beside the words then

Reply:
if for some reason we cannot get rid of the else statement, then another option to clean this up would be ternary statements:

```
let data = {};

data.show_tutorial = 0 ? $('#dontShowTutorialCheckbox').is(':checked') : data.show_tutorial = 1;
```